package.json

```diff
@@ -55,16 +55,11 @@
     "bootstrap": "node ./scripts/bootstrap.js",
     "build": "node ./scripts/build-package.js",
     "build-manager": "node -r esm ./scripts/build-manager.js",
-    "build-packs": "lerna exec --scope '@storybook/*' -- \\$LERNA_ROOT_PATH/scripts/build-pack.sh \\$LERNA_ROOT_PATH/packs",
```

Comment:
I think we used this in the past before we had `verdaccio`?

I'm fairly sure this hasn't been used in years.